install.sh

```diff
@@ -201,6 +201,36 @@ ee_notice() {
 	echo
 }
 
+ee_promo() {
+	# Randomly select an advertisement
+	echo
+	echo "** DOCKER ENGINE - ENTERPRISE **"
+	echo
+	case $(shuf -i 0-1 -n 1) in
+		0)
+			echo "Test drive additional security features by activating Docker Engine - Enterprise."
+			echo
+			echo "  * Leverage FIPS 140-2 validated encryption"
+			echo "  * Run only trusted images with digital signature enforcement"
+			;;
+		1)
+			echo "If you’re ready for production workloads, Docker Engine - Enterprise also includes:"
+			echo
+			echo "  * SLA-backed technical support"
+			echo "  * Extended lifecycle maintenance policy for patches and hotfixes"
+			echo "  * Access to certified ecosystem content"
+			;;
+	esac
+	echo
+	echo "** Learn more at https://www.docker.com/products/docker-engine **"
+	echo
+	echo "ACTIVATE your own engine to Docker Engine - Enterprise using:"
+	echo
+	echo "  sudo docker engine activate"
+	echo
+	sleep 10
```

Comment:
Is the sleep necessary?

Reply:
Necessary for "visibility"

Reply:
The sleep is followed by an exit so it would be the last thing seen before returning to a prompt, and thus left on screen unless the user did some kind of reset or clear in sequence.